src/server/resolvers.ts

```diff
@@ -64,7 +65,7 @@ async function query<T>(filter: FilterQuery<T>, model: Collection<T>): Promise<T
 	if (!obj)
 		throw new UserInputError(
 			`obj with filters: "${JSON.stringify(filter)}" not found in collection "${
-				model.collectionName
+			model.collectionName
```

Comment:
this breaks lint

Reply:
get this. fix it

Reply:
when I save it would automatically trigger prettier to remove that indent. pass this by add //prettier-ignore on top of that statement
not sure that is good QAQ

Reply:
*not sure if that is good

Reply:
don't do prettier-ignore here, disable editor format on save

Reply:
I pull-requested a change #334 that should fix your format on save